Makefile

```diff
@@ -44,14 +42,12 @@ build-local:
 
 push:
 	@$(CONTAINER_ENGINE) push $(CONSOLE_IMAGE):latest
-	@$(CONTAINER_ENGINE) push $(CONSOLE_IMAGE):${VERSION}
 	@$(CONTAINER_ENGINE) push $(CONSOLE_IMAGE):${IMAGE_TAG}
 
-
-start-dev: export REACT_APP_CIQ_API_URL = http://localhost:8081
+start-dev: export VITE_CIQ_API_URL = http://localhost:8081
 start-dev:
 	@echo "### [Starting project DEV MODE] ###"
-	@echo "### [API-URL: $$REACT_APP_CIQ_API_URL] ###"
+	@echo "### [API-URL: $$VITE_CIQ_API_URL] ###"
 	@npm run start
 
```

Comment:
Missing "stop-dev" command
```
stop-dev:
  podman rm -f console

```

Reply:
npm ERR! Missing script: "lint"
npm ERR! Missing script: "format"

When running: "make lint format"

Do we still using these commands?

Reply:
Answered at https://github.com/RHEcosystemAppEng/cluster-iq-console/pull/40#discussion_r1896079887